content/applications/websites/ecommerce/checkout_payment_shipping.rst

```diff
@@ -0,0 +1,10 @@
+:nosearch:
```

Comment:
I just copy/pasted an existing :toctree: and changed the files contained. No idea what :nosearch: does, but since it works like that for other :toctree:'s ... :)

Removed it.